stacker/lookups/output.py

```diff
@@ -0,0 +1,35 @@
+from collections import namedtuple
```

Comment:
ah i see what you mean. let me convert the KMS translator and lets see how it looks. just not sure what the right syntax should be since i've only done outputs so far